packages/host/app/components/ai-assistant/new-session.gts

```diff
@@ -114,6 +118,12 @@ export default class NewSession extends Component<Signature> {
       .error-section > * + * {
         margin-top: var(--boxel-sp-sm);
       }
+      small {
```

Comment:
I don’t know how big of a deal this is but the source of CSS scoping [warns](https://vue-loader.vuejs.org/guide/scoped-css.html#also-keep-in-mind) against only using tag names as selectors for performance reasons 🤔

Reply:
I was unsure about doing this, thanks